### ISSUE TITLE
Bug: Installation instructions unclear

### ISSUE THREAD

**Issue by ShalokShalom** [webosbrew/youtube-webos#180]
## YTAF information

- YTAF version: 0.3.4

## TV information

modelName : OLED55E97LA
sdkVersion : 4.10.0
firmwareVersion : 05.40.35
boardType : O18_DVB_EU
otaId : HE_DTV_W19O_AFABABAA

Hi there!

I found the instructions on the README to be incorrect for my installation process. 
Instead, the linked guide on the LG [homepage](https://webostv.developer.lge.com/develop/getting-started/developer-mode-app) helped me. 

In particular, entering the line described here was not sufficent to establish the passkey. 
![Screenshot_20240612_130722](https://github.com/webosbrew/youtube-webos/assets/6344099/d4610cb6-0ed8-4de0-bb22-0677ff1122d3)

Instead, it was needed to register it via `ares-novacom --device tv2 --getkey`

I suggest to double check, if the described method is still valid. 
I can rework the guide, once its confirmed that its not. 

The described method yielded the following issue: 

![Screenshot_20240612_125732](https://github.com/webosbrew/youtube-webos/assets/6344099/3d2a3927-f7b3-478f-a950-827a8f9dde69)

**Comment by throwaway96** [comment]
It appears `${HOME}/.ssh` (or `%USERPROFILE%\.ssh` on Windows) is prepended to the path given for private key files.

https://github.com/webos-tools/cli/blob/17f173a6a6263ed4110ecdb636fd1f33bfadc293/lib/base/novacom.js#L31

https://github.com/webos-tools/cli/blob/17f173a6a6263ed4110ecdb636fd1f33bfadc293/lib/base/novacom.js#L200

**Comment by ShalokShalom** [comment]
If you want, I did create a complete, step by step tutorial for unrooted devices. 

I found all the existing tutorials to lack certain information, and this is the only one that I know, which includes all the steps on one page. 

It also does not need any ssh key to be saved anywhere. 

https://gist.github.com/ShalokShalom/ba7bbeaa9c3fa7d1e12af77bcca05317

I would like to send you another PR, that does include all that information, and also rearranges the sections of the README so they make more sense.
